tasks/task968_xcopa_commonsense_reasoning_et.json

```diff
@@ -0,0 +1,3623 @@
+{
+    "Contributors": [
+        "Amirreza Mirzaei"
+    ],
+    "Source": [
+        "XCOPA (\"https://arxiv.org/abs/2005.00333\")"
+    ],
+    "Categories": [
+        "Answer Generation"
+    ],
+    "Definition": "In this task you are given a premise and two alternatives in Estonian you must choose the alternative that is more plausibly the cause or effect of the situation described by the premise. The input format is \"premise\n(1)alternative_1(2)alternative_2\", the output should either be \"1\" or \"2\" based on your judgment.",
+    "Input_language": "et",
+    "Output_language": "en",
+    "Instruction_language": "en",
+    "Positive Examples": [
+        {
+            "input": "Ese oli mullikilesse mässitud.\n(1)See oli õrn.(2)See oli väike.",
+            "output": "1",
+            "explanation": "The first alternative makes more sense because you put object is bubble wrap if they are delicate."
+        },
+        {
+            "input": "Ma tühjendasin oma taskud.\n(1)Ma leidsin pileti tüki.(2)Ma leidsin relva.",
+            "output": "1",
+            "explanation": "The second alternative makes more sense because finding a ticket in your pocket is more common than finding a gun."
```

Comment:
The output is "1" but in the explanation you have mentioned "The second alternative makes more sense"